package.json

```diff
@@ -36,10 +36,7 @@
     "@uiw/react-color-material": "^0.1.0",
     "@uiw/react-color-saturation": "0.0.34",
     "apollo": "^2.32.5",
```

Comment:
What about this one?

Reply:
Newest version is `2.33`

Reply:
Are we still using `apollo` package though?